build.gradle

```diff
@@ -170,7 +170,15 @@ dependencies {
     implementation "com.offbytwo.jenkins:jenkins-client:0.3.8"
     implementation "org.gitlab4j:gitlab4j-api:5.0.1"
 
-    implementation "de.jplag:jplag:3.0.0"
+    implementation "de.jplag:jplag:${jplag_version}"
+    implementation "de.jplag:languages:${jplag_version}"
```

Comment:
I guess the languages dependency is not needed. This is just a parent for the pre-defined languages of JPlag (but it's possible to keep that here, of course)